README/WithIn.agda

```diff
@@ -1,21 +1,20 @@
 ------------------------------------------------------------------------
 -- The Agda standard library
 --
--- Explaining how to use the inspect idiom and elaborating on the way
--- it is implemented in the standard library.
+-- Explaining how to use the `with ... in ...` idiom.
 ------------------------------------------------------------------------
```

Comment:
The standard library is not the right place to be writing tutorials on language features. I think this file should be kept as is (i.e. no changes) but marked deprecated just like the `inspect` function itself.

Reply:
Ah... `inspect` as a *definition* merited a a `README`, but `with ... in` as a language feature does not? Fair enough, I suppose. But as each file showed/shows, 'simple' (short!) examples are not so easy to come up with.

Reply:
When deprecating `README.Inspect`, is it necessary to deprecate each of the definitions in the module, or only the module (name) itself at the top? I presume the latter, as nothing can (or should!)  `import README.Inspect`, except `README.agda` itself. But will the deprecation warning then trigger a failing `make test`?

Reply:
I have removed the import of `README.Inspect` from `README`. Let me know if this is a problem!